cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/KafkaCruiseControl.java

```diff
@@ -120,7 +121,8 @@ public void run() {
                                                            boolean throttleDecommissionedBroker,
                                                            List<String> goals,
                                                            ModelCompletenessRequirements requirements,
-                                                           OperationProgress operationProgress)
+                                                           OperationProgress operationProgress,
+                                                           boolean isKafkaAssignerMode)
```

Comment:
Can we avoid exposing this to the public interface? Kafka assigner mode is essentially a combination of goals defined in the servlet. Cruise Control should not be aware of this.